### PR TITLE
Visualisations in bars and heatmap alls items and action type interactions 

### PR DESCRIPTION
**Pull request by gorco**

### CODE REVIEW

**Comment by gorco** [comment]
#5 substitute

**Review by jujusb — APPROVED** [review]
After the correction of a file problem, it's all OK

File: vis/xAPISG-ItemsActionTypeInteracted.ipynb [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+   "source": [
+    "def vis_items_action_type_interacted_times(players_info):\n",
+    "    # find full list of items\n",
+    "    items=set()\n",
+    "    for player in players_info.keys():\n",
+    "        for type in players_info[player][\"action_type_interaction\"].keys():\n",
+    "            if players_info[player][\"action_type_interaction\"][type] != {}:                \n",
+    "                items = items.union(players_info[player][\"action_type_interaction\"][type].keys())\n",
+    "    items = list(items)\n",
+    "    #print(items)\n",
```
Comment (crisal24) [inline vis/xAPISG-ItemsActionTypeInteracted.ipynb]: Delete commented lines

File: vis/xAPISG-ItemsActionTypeInteracted.ipynb [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    "    number_of_plots = len(nbitems)\n",
+    "    xfigsize=len(players_info.keys())\n",
+    "    yfigsize=number_of_plots*5\n",
+    "    #print(xfigsize)\n",
+    "    #print(yfigsize)\n",
+    "    if xfigsize!=0 and number_of_plots>1:\n",
+    "        fig, axs = plt.subplots(number_of_plots, figsize=(xfigsize,yfigsize), sharex=True)\n",
+    "        i = 0\n",
+    "        for item in nbitems.keys():\n",
+    "            j=0\n",
```
Comment (crisal24) [inline vis/xAPISG-ItemsActionTypeInteracted.ipynb]: Variable not updated/used

File: vis/xAPISG-ItemsInteracted.ipynb [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "def find_list_item(players_info): \n",
```
Comment (crisal24) [inline vis/xAPISG-ItemsInteracted.ipynb]: In xAPISG-ItemsActionTypeInteracted similar functionality is performed directly in the main function, please unify (either create def also in the other notebook, or delete this here).

File: vis/HelpersFunctions/HeatMap.ipynb [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+  {
+   "cell_type": "markdown",
+   "source": [
+    "***constructHeatMap* function** to constuct the multilist for the heatmap\n",
+    "\n",
+    "Input :\n",
+    "* players_info dict with players\n",
+    "* labels : labels for the construction\n",
+    "* action : the action to have to search in players_info\n",
+    "* typ : boolean if this action is divided in type or not\n",
```
Comment (crisal24) [inline vis/HelpersFunctions/HeatMap.ipynb]: Clarify / further document these variables for better understanding

File: vis/HelpersFunctions/MultiBarSeparated.ipynb [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+    }
+   },
+   "source": [
+    "***constructMultiTab* function** to constuct the multilist for the heatmap\n",
+    "\n",
+    "Input :\n",
+    "* players_info dict with players\n",
+    "* labels : labels for the construction\n",
+    "* action : the action to have to search in players_info\n",
+    "* typ : boolean if this action is divided in type or not\n",
```
Comment (crisal24) [inline vis/HelpersFunctions/MultiBarSeparated.ipynb]: Clarify / further document these variables for better understanding

File: xAPISGProcessor.ipynb [hunk 6]
```diff
@@ -354,6 +354,56 @@
...
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "%run vis/HelpersFunctions/MultiBarSeparated.ipynb\n",
```
Comment (crisal24) [inline xAPISGProcessor.ipynb]: Helper functions could better be run in the visualization notebook, to hide lower-level details here.

File: xAPISGProcessor.ipynb [hunk 7]
```diff
@@ -354,6 +354,56 @@
...
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "%run vis/HelpersFunctions/HeatMap.ipynb\n",
```
Comment (crisal24) [inline xAPISGProcessor.ipynb]: Helper functions could better be run in the visualization notebook, to hide lower-level details here.

File: xAPISGProcessor.ipynb [hunk 8]
```diff
@@ -354,6 +354,56 @@
...
+    "%run vis/xAPISG-ItemsInteracted.ipynb\n",
```
Comment (crisal24) [inline xAPISGProcessor.ipynb]: Duplicated: already run in previous cell.

File: xAPISGProcessor.ipynb [hunk 9]
```diff
@@ -354,6 +354,56 @@
...
+    "vis_items_interacted_times_heatMap(players_info)"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
   }
  ],
```
Comment (crisal24) [inline xAPISGProcessor.ipynb]: Vis notebook xAPISG-ItemsActionTypeInteracted is not used.

**Review by crisal24 — CHANGES_REQUESTED** [review]
Good job! Some minor issues to fix.

**Pull request closed without merging**